src/hnsw/external_index.c

```diff
@@ -16,6 +16,9 @@
 
 static inline void  fill_blockno_mapping();
 static BlockNumber *wal_retriever_block_numbers = NULL;
+static uint32 g_blockmap_groups = 0;
+static uint32 g_blockmap_group_indexs[ 32 ];
```

Comment:
indexs -> indexes 
`g_blockmap_group_indexes`

Reply:
and the constant should be `HNSW_MAX_BLOCKMAP_GROUPS` right?

Reply:
yep